charts/docker-registry-ui/README.md

```diff
@@ -121,6 +121,10 @@ helm upgrade --install docker-registry-ui joxit/docker-registry-ui
 | `registry.annotations` | `{}` | Annotations to apply to the registry server deployment. |
 | `registry.additionalSpec` | `{}` | Optional YAML string that will be appended to the deployment spec. |
 | `registry.extraEnv` | `[]` | Extra Environmental Variables for Registry |
+| `registry.basicAuth.enabled` | `false` | Enable basic auth for Registry.|
+| `registry.basicAuth.realm` | `Docker registry` | Basic auth realm.|
+| `registry.basicAuth.htpasswdPath` | `/auth/htpasswd` | Full path for htpasswd file. Note that filename should match the secret key.|
+| `registry.basicAuth.secretName` | `` | Basic Auth (htpasswd) Secret Name for Registry.|
```

Comment:
The README is managed by the CI and in house documentation generator

https://github.com/Joxit/helm-charts/blob/772be0269aa8d1b167a5b1190665a36916cf8692/.github/workflows/release.yaml#L20-L22